rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -600,6 +601,43 @@ http {
 }
 
 stream {
+    {{ if $all.IsSSLPassthroughEnabled }}
+    # map FQDN that requires SSL passthrough
+    map $ssl_preread_server_name $stream_upstream {
+        hostnames;
+        {{ range $i, $passthrough := .PassthroughBackends }}
+        {{   $passthrough.Hostname }}         {{  $passthrough.Backend }};
+        {{ end }}
+
+        # Add non SSLPassthrough backends list to gracefully handle situation when SSLPassthrough wildcard overlaps with non SSLPassthrough direct matching domain
+        {{ range $server := $servers }}
+        {{   if $server.SSLPassthrough }}
+        {{     $server.Hostname }} nginx-ssl-backend;
+        {{     range $server.Aliases }}
+        {{       . }} nginx-ssl-backend;
+        {{     end }}
+        {{   end }}
+        {{ end }}
+
+        # send SSL traffic to this nginx in a different port
+        default                         nginx-ssl-backend;
+    }
+
+    # configure default backend for SSL
+    upstream nginx-ssl-backend {
+        server 127.0.0.1:{{ $all.ListenPorts.SSLProxy }};
+    }
+
+    {{ buildSSLPassthroughUpstreams $backends .PassthroughBackends }}
+
+    server {
+        listen                  443 {{ if $cfg.UseProxyProtocol }} proxy_protocol{{ end }};
```

Comment:
Use `{{ $all.ListenPorts.HTTPS }}` instead of 443

Reply:
Ready